docs/developer.md

```diff
@@ -253,3 +253,45 @@ cobra add commandName
 ```
 
 You should see a new file appear under `./cmd/armadactl/cmd` with the name you specified in the command.
+
+### Setting up OIDC for developers.
+
+Setting up OIDC can be an art.  We found that [Okta Developer Program](https://developer.okta.com/signup/) provides a nice quick-and-dirty way to test OAuth flow.
+
+1) Create a Okta Developer Account
+    - I used my github account.
+2) Create a new App in the Okta UI.
+    - Select OIDC - OpenID Connect.
+    - Select Web Application.
+3) In grant type, make sure to select Client Credentials.  We tested this one out because it requires little interaction from the browser.  
```

Comment:
How about changing "We tested this one out because it requires little interaction" to "This has the advantage of requiring little interaction"?  (Mostly to keep consistent the "active" vs. "passive" voice.)